.github/workflows/test.yaml

```diff
@@ -109,11 +108,6 @@ jobs:
           else
             echo "No solr directory found, skipping..."
           fi
-      - name: Setup db
```

Comment:
maybe an if statement for this one?

Reply:
I'd need to check for the existence of a database